docs/blog/newsletter-november-2022.rst

```diff
@@ -0,0 +1,131 @@
+
+Many people advised spending time getting to know the team, team members' strengths and weaknesses, processes, existing docs, and where the team fits in the overall organization. One person suggested extending that to focus on not making any changes at all for a while. Take the time to understand the entire context and all expectations from the team and the rest of the organization before committing to anything new.
+
+As you're evaluating the context, consider where you need to form relationships around the organization to help the team work effectively. Find out what business goals the team helps with and what needs other teams have that the documentation team could help with. Learning this context helps you determine how you can best advocate for your team.
+
+As you learn how to advocate for your team, think about how to grow it to meet the needs you discovered. This can mean helping people with professional development or figuring out who can be hired and how to get the budget for that. Creating a long-term plan based on what you determine in the first few months can help you place your team needs in a context that will help get them addressed.
+
+----------------------------
+Minimum viable documentation
+----------------------------
+
+Minimum viable documentation is the minimum required information that's usable and helpful to users. For many reasons, it’s seen positively by some and adversely by others. Those who support it say it’s the best place to start if a product lacks any documentation. Additionally, in places where releases are frequent, minimal documentation enable you to write the essential information and then return to the document to update it once things have stabilized. But there are competing views on minimal documentation, and those who work at places without it believe that the "document everything" approach can lead to many issues, including leaning more towards a mindset of "done is better than perfect."
+
+Proponents of minimal documentation consider that a product should be capable of functioning with little explanation from its authors. They say this provides a quicker turnaround time on new features, enables faster bug fixes, and simplifies developers’ lives by removing tiresome writing duties. Opponents claim that this strategy results in varying quality across products and can lead to knowledge gaps within teams. They argue that all software should be documented since it eliminates uncertainty when using new systems (and makes life easier for users).
+
+If you’re interested in learning more about minimal viable documentation, read this `blog post from Sarah Moir <https://thisisimportant.net/posts/from-nothing-to-something-with-minimum-viable-documentation/>`__. It covers defining such documentation, assessing documentation states, and structuring documentation.
```

Comment:
I noticed this item switches between `minimum` and `minimal` -- I'm not sure if there's a distinction made in the original conversation but it seems like they're being used interchangeably here.

Reply:
That's probably just my editing mistake. Thanks for the note!